### PR TITLE
[MIG] [17.0] web_groupby_expand: migrated module

### PR DESCRIPTION
**Pull request by ShwetaSerpentCS**
Responsible : Adnan

### CODE REVIEW

File: web_groupby_expand/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "author": "Serpent Consulting Services Pvt. Ltd.",
+    "maintainer": "Serpent Consulting Services Pvt. Ltd.",
+    "website": "http://www.serpentcs.com",
+    "category": "Web",
+    "license": "AGPL-3",
+    'summary': 'Expand all groups on single click',
+    "depends": [
+        "web",
+    ],
+    "data": [
```
Comment (Trivedi-Vacha-SerpentCS) [inline web_groupby_expand/__manifest__.py]: Remove the blank keys

**Review by Trivedi-Vacha-SerpentCS — CHANGES_REQUESTED** [review]

**Comment by ketanserpentcs** [comment]
Testing is assigned to Pooja

**Comment by ShwetaSerpentCS** [comment]
QA done by Pooja.

**Comment by ketanserpentcs** [comment]
QA done with index by Pooja

**Pull request merged**